public/507.html

```diff
@@ -0,0 +1,143 @@
+  </head>
+	<body>
+		<ul id="skiplinks"><li><a href="#main">Main Content</a></li></ul>
+    <!-- BEGIN header -->
+    <div id="header" class="region">
+      <h1 class="heading">
+        <a href="/"><span>Archive of Our Own</span><sup> beta</sup><img alt="AO3 logo - the letters A O 3 combined with arms raised in celebration, symbolizing the joy of fannish creation in the Archive" class="logo" src="/images/ao3_logos/logo_42.png?1362864128" /></a> 
+      </h1>
+
+      <h3 class="landmark heading">Site Navigation</h3>
+      <ul class="primary navigation actions" role="navigation">
+        <li class="dropdown">
+          <a href="/menu/fandoms">Fandoms</a>
+          <ul class="menu" role="menu">
+            <li><a href="/media">All Fandoms</a></li>
+            <li id="medium_5"><a href="/media/Anime%20*a*%20Manga/fandoms">Anime &amp; Manga</a></li>
+            <li id="medium_3"><a href="/media/Books%20*a*%20Literature/fandoms">Books &amp; Literature</a></li>
+            <li id="medium_4"><a href="/media/Cartoons%20*a*%20Comics%20*a*%20Graphic%20Novels/fandoms">Cartoons &amp; Comics &amp; Graphic Novels</a></li>
+            <li id="medium_7"><a href="/media/Celebrities%20*a*%20Real%20People/fandoms">Celebrities &amp; Real People</a></li>
+            <li id="medium_2"><a href="/media/Movies/fandoms">Movies</a></li>
+            <li id="medium_6"><a href="/media/Music%20*a*%20Bands/fandoms">Music &amp; Bands</a></li>
+            <li id="medium_8"><a href="/media/Other%20Media/fandoms">Other Media</a></li>
+            <li id="medium_30198"><a href="/media/Theater/fandoms">Theater</a></li>
+            <li id="medium_1"><a href="/media/TV%20Shows/fandoms">TV Shows</a></li>
+            <li id="medium_476"><a href="/media/Video%20Games/fandoms">Video Games</a></li>
+            <li id="medium_9971"><a href="/media/Uncategorized%20Fandoms/fandoms">Uncategorized Fandoms</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/browse">Browse</a>
+          <ul class="menu" role="menu">
+            <li><a href="/works">Works</a></li>
+            <li><a href="/bookmarks">Bookmarks</a></li>
+            <li><a href="/tags">Tags</a></li>
+            <li><a href="/collections">Collections</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/search">Search</a>
+          <ul class="menu" role="menu">
+            <li><a href="/works/search">Works</a></li>
+            <li><a href="/people/search">People</a></li>
+            <li><a href="/bookmarks/search">Bookmarks</a></li>
+            <li><a href="/tags/search">Tags</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/about">About</a>
+          <ul class="menu" role="menu">
+            <li><a href="/about">About Us</a></li>
+            <li><a href="/admin_posts">News</a></li>
+            <li><a href="/archive_faqs">FAQ</a></li>
```

Comment:
This also needs the Wrangling Guidelines part (`<li><a href="/wrangling_guidelines">Wrangling Guidelines</a></li>`) added after the FAQ.